src/sagemaker/feature_store/feature_group.py

```diff
@@ -285,13 +297,21 @@ def init_worker():
         self._processing_pool = ProcessingPool(self.max_processes, init_worker)
         self._processing_pool.restart(force=True)
 
-        f = lambda x: self._run_multi_threaded(*x)  # noqa: E731
+        f = lambda x: IngestionManagerPandas._run_multi_threaded(*x)  # noqa: E731
         self._async_result = self._processing_pool.amap(f, args)
 
         if wait:
             self.wait(timeout=timeout)
 
-    def _run_multi_threaded(self, data_frame: DataFrame, row_offset=0, timeout=None) -> List[int]:
+    @staticmethod
```

Comment:
@verayu43 have you tested this change on `py38` and `py39` ?

Reply:
I used the fraud detection example script reproduced the failure in py39, and after this fix the example script can run correctly in py39.

Reply:
Hi Ahsan, the code change test successfully by running "tox -e py38 tests/unit" and "tox -e py39 tests/unit". Thanks!